torch/csrc/jit/tracer.cpp

```diff
@@ -200,6 +200,14 @@ Value* TracingState::getOutput(const IValue& iv, size_t i) {
        throw std::runtime_error(os.str());
      }
      return it->second;
+  } else if (iv.isTensorList()) {
```

Comment:
you should also edit the error message to reflect the fact that list of tensor can be output of traced functions.